common/src/main/java/org/apache/uniffle/common/compression/ZstdCodec.java

```diff
@@ -69,7 +76,16 @@ public void decompress(ByteBuffer src, int uncompressedLen, ByteBuffer dst, int
 
   @Override
   public byte[] compress(byte[] src) {
-    return Zstd.compress(src, compressionLevel);
+    ZstdCompressCtx ctx = new ZstdCompressCtx();
```

Comment:
One question: Do we need to create Ctx for every time? It will cost time to create and destroy the thread. If ZstdCompressCtx will create and destroy the thread.

Reply:
Nice catch. Actually the code of `Zstd.compress(src, compressionLevel); ` also will create ctx everytime. Let me dig this deeply

Reply:
It's ok if the default context don't need to create extra threads actually. Just feel that we should consider more about this place.

Reply:
Yes, the compression ctx could be reused.